.github/workflows/build.yml

```diff
@@ -391,9 +391,10 @@ jobs:
           path: coverage
 
       - name: Codecov
-        uses: codecov/codecov-action@ab904c41d6ece82784817410c45d8b8c02684457 # v3.1.6
+        uses: codecov/codecov-action@e0b68c6749509c5f83f984dd99a76a1c1a231044 # v4.0.1
         with:
           directory: coverage/lcov
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
@viceice

Is `CODECOV_TOKEN` available as the repository secret? Seems like actually it isn't, as I'm getting `Error: Codecov token not found. Please provide Codecov token with -t flag.` in logs.